### PR TITLE
Fix invalid option definitions causing crashes in the JIT

### PR DESCRIPTION
**Pull request by fjeremic**
The `TR::OptionTable::msg` field must be non-null, otherwise when
printing the options via the `verbose` command we run into null pointer
dereferences which causes the JIT to crash.

Fixes: #3955

Signed-off-by: Filip Jeremic <fjeremic@ca.ibm.com>

### CODE REVIEW

**Comment by fjeremic** [comment]
@dsouzai could I please get a review?

File: runtime/compiler/control/J9Options.cpp [hunk 1]
```diff
@@ -981,13 +981,13 @@ TR::OptionTable OMR::Options::_feOptions[] = {
    {"verbose=",           "L{regex}\tlist of verbose output to write to vlog or stdout",
         TR::Options::setVerboseBitsInJitPrivateConfig, offsetof(J9JITConfig, privateConfig), 0, "F"},
    {"version",            "L\tdisplay the jit build version",
-        TR::Options::versionOption, 0, 0},
+        TR::Options::versionOption, 0, 0, "F"},
```
Comment (dsouzai) [inline runtime/compiler/control/J9Options.cpp]: Isn't the second `0` supposed to be replaced with the `"F"?

File: runtime/compiler/control/J9Options.cpp [hunk 2]
```diff
@@ -981,13 +981,13 @@ TR::OptionTable OMR::Options::_feOptions[] = {
...
    {"veryHotSampleThreshold=",          "R<nnn>\tThe maximum number of global samples taken during a sample interval for which the method will be recompiled at hot with normal priority",
         TR::Options::setStaticNumeric, (intptrj_t)&TR::Options::_veryHotSampleThreshold, 0, " %d", NOT_IN_SUBSET},
    {"vlog=",              "L<filename>\twrite verbose output to filename",
         TR::Options::setString,  offsetof(J9JITConfig,vLogFileName), 0, "F%s"},
    {"vmState=",           "L<vmState>\tdecode a given vmState",
-        TR::Options::vmStateOption, 0, 0},
+        TR::Options::vmStateOption, 0, 0, "F"},
```
Comment (dsouzai) [inline runtime/compiler/control/J9Options.cpp]: Same comment as above.

File: runtime/compiler/control/J9Options.cpp (hunk 1 above)
Comment (fjeremic) [reply runtime/compiler/control/J9Options.cpp]: I don't think so. If you take a look at the `"verbose="` option above and the actual definition of `TR::OptionTable` [1] you will see the order is, `name`, `helpText`, `fcn`, `parm1`, `parm2`, and then `msg` which is not specified here (so assumed `NULL` - see [2] for a live example).

[1] https://github.com/eclipse/omr/blob/7cf542c14aa04cc5e5baafa4cd88a7b3fbef6f98/compiler/control/OptionsUtil.hpp#L48-L137
[2] https://ideone.com/03Ediv

**Comment by mstoodle** [comment]
A very safe fix and no platform specific aspect. Thanks @fjeremic !

**Pull request merged**